src/components/pie-chart/pie-chart.js

```diff
@@ -0,0 +1,33 @@
+import template from "./pie-chart.html";
+import Chart from "chart.js";
+
+export class PieChartComponent {
+  constructor(mountPoint) {
+    this.mountPoint = mountPoint;
+  }
+
+  querySelectors() {
+    this.pieChart = this.mountPoint.querySelector(".pie-chart__visual");
+  }
+
+  mount() {
+    this.mountPoint.innerHTML = template();
+    this.querySelectors();
+    this.createPieChart();
+  }
+
+  createPieChart() {
+    let myPieChart = new Chart(this.pieChart, {
```

Comment:
I would call it so: this.pieChartCtx, (instead this.pieChart) and pieChart (instead myPieChart). maybe you like it